src/styles/main.scss

```diff
@@ -1,3 +1,81 @@
+@import 'variables';
```

Comment:
I can't find your variables file, where is it?

Reply:
Hmm, for some reason it wasn't added, i will fix it, thanks.